### ISSUE TITLE
coverage_content_type

### ISSUE THREAD

**Issue by akrishnan1** [ioos/compliance-checker#591]
The ACDD 1.3 check currently does not validate the coverage_content_type attribute values against the controlled codelist at
https://geo-ide.noaa.gov/wiki/index.php?title=ISO_19115_and_19115-2_CodeList_Dictionaries#MD_CoverageContentTypeCode

Some of the files that I tested had the coverage_content_type listed as "auxiliaryInformation" and those still passed the check

**Comment by Bobfrat** [comment]
Great catch. We do actually check that but we probably created that list from here, where there is the same misspelling you're reporting:

http://wiki.esipfed.org/index.php/Attribute_Convention_for_Data_Discovery_1-3#coverage_content_type


For reference
https://github.com/ioos/compliance-checker/blob/master/compliance_checker/acdd.py#L699-L712

**Comment by Bobfrat** [comment]
Or is the misspelling is actually on the ISO end? I don't think auxillary is a word.

**Comment by akrishnan1** [comment]
That's right. How should this be resolved, then? Personally, I think we should keep it the way it has been spelled in the ISO document.

**Comment by Bobfrat** [comment]
I suppose it would be prudent to let someone know of the misspelling and inconsistency between ESIP site and the ISO document. Not sure exactly who to talk to. @jbosch-noaa any ideas?

One option is we could accept both spellings in the compliance checker.

**Comment by akrishnan1** [comment]
I have sent an email to the ESIP mailing list. Will wait for folks to weigh in.

**Comment by mwengren** [comment]
Maybe @amilan17 has some insight into it.  Most NOAA metadata seems to refer to the ISO Codelists hosted at NCEI (formerly NGDC at these URLs: https://www.ngdc.noaa.gov/emma/xsd/schema/resources/Codelist/gmxCodelists.xml#MD_CoverageContentTypeCode ) as linked from the GEO-IDE wiki, but I know these are in the process of moving to a new home/URL.  

But perhaps this error is in the official ISO codelists as well, not just the NOAA ones, I honestly don't know where to find those offhand.

**Comment by akrishnan1** [comment]
I found an official copy of the ISO 19115-1 document and I see the typo in it as well.

**Comment by mwengren** [comment]
Maybe @amilan17 has some insight into it.  Most NOAA metadata seems to refer to the ISO Codelists hosted at NCEI (formerly NGDC at these URLs: https://www.ngdc.noaa.gov/emma/xsd/schema/resources/Codelist/gmxCodelists.xml#MD_CoverageContentTypeCode ) as linked from the GEO-IDE wiki, but I know these are in the process of moving to a new home/URL.  

But perhaps this error is in the official ISO 19115-1 codelists as well, and reflected in the GEO-IDE [wiki link](https://geo-ide.noaa.gov/wiki/index.php?title=ISO_19115_and_19115-2_CodeList_Dictionaries#MD_CoverageContentTypeCode) you posted.

**Comment by benjwadams** [comment]
Have any conclusions been reached regarding the accepted values?
